tests/acceptance/30_generic_methods/unsafe/service_check_disabled_at_boot.cf

```diff
@@ -0,0 +1,53 @@
+#######################################################
+#
+# Test if cron is started and if yes, stop it
```

Comment:
Isn't this comment misleading ?
It does ensure that cron is disabled at boot, and check then that it is disabled